packages/components/organisms/f-checkout/src/components/_tests/Selector.test.js

```diff
@@ -496,6 +496,55 @@ describe('Selector', () => {
                     expect(wrapper.vm.selectedAvailableFulfilmentTime).toBe('');
                 });
             });
+
+            describe('when a future fulfilment time has been selected', () => {
+                // eslint-disable-next-line prefer-destructuring
+                const selectedTime = fulfilmentTimes[1];
+                const asapTime = fulfilmentTimes[0].from;
+                const laterTime = fulfilmentTimes[1].from;
+
+                describe('AND `hasAsapSelected` is true', () => {
+                    beforeEach(() => {
```

Comment:
As you have only one it statement there is no need for beforeEach here. The same is for line 528